### PR TITLE
sketchybar: new port

### PR DESCRIPTION
**Pull request by bashu**
#### Description

Custom macOS statusbar with shell plugin, interaction and graph support

###### Type(s)

- [x] enhancement

###### Tested on

macOS 12.6.1 21G217 x86_64
Xcode 14.1 14B47b

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] checked your Portfile with `port lint --nitpick`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by bashu** [comment]
I skipped: `port test ...` cuz there are no tests for this port

File: sysutils/sketchybar/Portfile [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+PortSystem              1.0
+PortGroup               github 1.0
+PortGroup               makefile 1.0
+PortGroup               xcodeversion 1.0
+
+github.setup            FelixKratz SketchyBar 2.11.1 v
+
+categories              sysutils
+platforms               darwin
```
Comment (herbygillot) [inline sysutils/sketchybar/Portfile:11]: `platforms darwin` is now implicit, so you can remove this line

**Comment by herbygillot** [comment]
This looks good @bashu, just one comment (above).

File: sysutils/sketchybar/Portfile (hunk 1 above)
Comment (bashu) [reply sysutils/sketchybar/Portfile]: fixed

**Review by herbygillot — APPROVED** [review]

**Comment by herbygillot** [comment]
Thanks for your submission @bashu

**Pull request merged**